test/powershell/engine/Basic/Assembly.LoadWithPartialName.Tests.ps1

```diff
@@ -19,7 +21,7 @@ Describe "Assembly::LoadWithPartialName Validation Test" -Tags "CI" {
 
     # All existing cases should fail on all platforms either because it doesn't exist or
     # because the assembly is blacklisted
-    It "Assembly::LoadWithPartialName should fail to load blacklisted assembly: <Name>" -TestCases $testcases {
+    It "Assembly::LoadWithPartialName should fail to load blacklisted assembly: <Name>" -Skip -TestCases $testcases {
```

Comment:
We need comment why we skip this. (Should it be pending?)

Reply:
Pending means there is a problem with the test, in this case the test is fine, but there's nothing being blacklisted at the moment.  I'll add a comment.